packages/headless/src/api/service/insight/user-actions/user-actions-response.ts

```diff
@@ -1,23 +1,41 @@
+export enum UserActionType {
```

Comment:
I guess we'll need to add another type for "Ticket created".

Reply:
Yes good point